index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title> my portfolio</title>
+</head>
+
+<body>
+    <main class="container">
+        <section>
+            <a class="logo">logos</a>
+            <nav class="humbuger"><img src="images/icones/Union.png" alt="humburger"> </nav>
+        </section>
+        <div class="contain">
+            <div class="container-title">
+                <h1 class="item1">Faustin Makutano</h1>
+                <h2 class="item2">Software Developer</h2>
+                <div class="literature">
+                    <p class="item3">I can help you build a product , feature or website Look through some of my work and experience!<br> If you like what you see and have a project you need coded,<br> don’t hestiate to contact me.</p>
+                </div>
+            </div>
+        </div>
+        <header class="border-bottom">
+            <ul class="skills-list">
+                <li class="language">
+                    <h3 class="skill-title">Language</h3>
+                    <ul class="sublist">
+                        <li class="skill">JavaScript</li>
+                        <li class="skill">Ruby</li>
+                        <li class="skill">Html</li>
+                        <li class="skill">Css</li>
+                    </ul>
+                </li>
+                <li class="frameworks">
+                    <h3 class="skill-title">Frameworks</h3>
+                    <ul class="sublist">
+                        <li class="skill">Bootstrap</li>
+                        <li class="skill">Ruby on Rails</li>
+                        <li class="skill">RSPec</li>
+                        <li class="skill">Capy Bary</li>
+                        <li class="skill">Selenium</li>
+                    </ul>
+                </li>
+                <li class="skills">
+                    <h3 class="skill-title">Skills</h3>
+                    <ul class="sublist">
+                        <li class="skill">Codekit</li>
+                        <li class="skill">Github</li>
+                        <li class="skill">Codpen</li>
+                        <li class="skill">Gitlab</li>
+                        <li class="skill">Terminal</li>
+                    </ul>
+                </li>
+            </ul>
+        </header>
+        <hr/>
+        <ul class="icones">
+            <li><img src="images/icones/sector.png" alt="icon1"></li>
+            <li><img src="images/icones/Vecto.png" alt=" icon2"></li>
+            <li><img src="images/icones/Vector.png" alt="icon3"></li>
+            <li><img src="images/icones/Vector.svg" alt="icon4"></li>
+            <li><img src="images/icones/Linkedin.png" alt="linkedinicon"></li>
+        </ul>
```

Comment:
- [ ] You should wrap the images with `<a>` tag so that they become clickable and help the visitor navigate to your social media profiles.
- [ ] You should also correct the vertical alignment of the icons.
![image](https://user-images.githubusercontent.com/44057863/170761921-3eab2af3-743c-4e4a-b9b7-d85e1a20b718.png)

- [ Optional ] You may use Fontawesome icons instead of images as well. please refer [this](https://fontawesome.com/docs/web/add-icons/how-to) on how to add `fa icons`.